lib/chef/audit/runner.rb

```diff
@@ -79,11 +79,15 @@ def require_deps
         require 'rspec'
         require 'rspec/its'
         require 'specinfra'
+        require 'specinfra/helper'
+        require 'specinfra/helper/set'
         require 'serverspec/helper'
         require 'serverspec/matcher'
         require 'serverspec/subject'
         require 'chef/audit/audit_event_proxy'
         require 'chef/audit/rspec_formatter'
+
+        Specinfra::Backend::Cmd.send(:include, Specinfra::Helper::Set)
```

Comment:
Agree, this is just fancy monkey patching of a dep, but maybe Specinfra is designed to work this way?